### PR TITLE
README intro writing improvements

### PR DESCRIPTION
**Pull request by eugenividal**
@Robinlovelace @e-kotov 

Hi both, this PR includes some little improvements to the README introduction. The changes provide more details on the Spanish OD data and explain how the data is organised on the website. Please approve only if you find it clearer. Feel free also to amend my own changes. I hope it helps!

### CODE REVIEW

File: Documents/sapsnioddata_test/clean_data/v2/zones/distritos_mitma.gpkg
Comment (Robinlovelace) [inline Documents/sapsnioddata_test/clean_data/v2/zones/distritos_mitma.gpkg:1]: His shouldn't be in here.

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
Can you put in a new PR without the files in the Documents folder Eugeni?

**Comment by Robinlovelace** [comment]
I've closed the PR and deleted this branch to avoid making this repo 'heavy' with the 25 MB file. May be quickest to make changes just to the README.qmd file in another lighter PR.

**Comment by eugenividal** [comment]
Sorry! Let me see if I manage to send the PR only with the README.

**Pull request closed without merging**